db/re/status.yml

```diff
@@ -1125,7 +1125,6 @@ Body:
       Shadowweapon: true
       Ghostweapon: true
   - Status: Volcano
-    Icon: EFST_GROUNDMAGIC
```

Comment:
These also have an EFST by the way, but it's called EFST_SPECIALZONE. It includes the element in the val.